src/jfem.jl

```diff
@@ -88,11 +85,12 @@ please refer to http://matplotlib.org/api/pyplot_api.html
 not all kwargs have been imported. Should you require any that are not imported
 open as issue, and I will attempt to add them.
 """
-Plot(in_plot::Union{Mesh,FunctionSpace};alpha=1,animated=false,antialiased=true,color="grey"
+Plot(in_plot::Union{Mesh,FunctionSpace,Function};alpha=1,animated=false,antialiased=true,color="grey"
 ,dash_capstyle="butt",dash_joinstyle="miter",dashes="",drawstyle="default",fillstyle="full",label="s",linestyle="solid",linewidth=1
 ,marker="",markeredgecolor="grey",markeredgewidth="",markerfacecolor="grey"
 ,markerfacecoloralt="grey",markersize=1,markevery="none",visible=true,title="") =fenics.common[:plotting][:plot](in_plot.pyobject,
 alpha=alpha,animated=animated,antialiased=antialiased,color=color,dash_capstyle=dash_capstyle,dash_joinstyle=dash_joinstyle
 ,dashes=dashes,drawstyle=drawstyle,fillstyle=fillstyle,label=label,linestyle=linestyle,linewidth=linewidth,marker=marker,markeredgecolor=markeredgecolor
 ,markeredgewidth=markeredgewidth,markerfacecolor=markerfacecolor,markerfacecoloralt=markerfacecoloralt,markersize=markersize,markevery=markevery
 ,visible=visible,title=title)#the first is the keyword argument, the second is the value
+export Plot
```

Comment:
exporting Plot is dangerous and will run into issues with many libraries. We should not do this. You should keep this documented but unexported, and the "easy" plotting functionality can come through recipes later.